src/Aspire.Dashboard/Components/Controls/Chart/MetricTable.razor.cs

```diff
@@ -31,6 +32,14 @@ public partial class MetricTable : ChartBase
 
     protected override async Task OnChartUpdated(List<ChartTrace> traces, List<DateTimeOffset> xValues, bool tickUpdate, DateTimeOffset inProgressDataTime)
     {
+        // Only update table every second
+        if (inProgressDataTime - _lastUpdate < TimeSpan.FromSeconds(1))
```

Comment:
I don't know the code. If another update never comes in, does it mean that the data before the update we skipped here will display indefinitely? In other words, should this update be deferred rather than ignored?

Reply:
> I think this logic is just evaluating how often to refresh the values (like a refresh interval)

Yep! No data is lost; the instrument updates its data every [200ms](https://github.com/dotnet/aspire/blob/88cece37d28245548e21735fe8a739a2e0ed7239/src/Aspire.Dashboard/Components/Controls/Chart/ChartContainer.razor.cs#L91) after which this method is called.